src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
+    public static String getRandomColor() {
```

Comment:
Avoid using static methods for functionality like getting a random color. It's better to create an instance of `ColorSupplier` and use instance methods.